README.md

```diff
@@ -195,6 +196,25 @@ class MyCommands implements CustomEventAwareInterface
 ```
 It is up to the command that defines the custom event to declare what the expected parameters for the callback function should be, and what the return value is and how it should be used.
 
+### Replace Command Hook
+
+The replace-command ([ReplaceCommandHookInterface](src/Hooks/ReplaceCommandHookInterface.php)) hook permits you to replace a command's method with another method of your own.
+
+For instance, if you'd like to replace the `foo:bar` command, you could utilize the following code:
+
+```php
+<?php
+class MyReplaceCommandHook  {
+
+  /**
+   * @hook replace-command foo:bar
+   */
+  public function myFooBarReplacement($args) {
```

Comment:
Actually, I had to modify things so that they are all passed as a single array of arguments.